ansible/roles/caddy/templates/Caddyfile.j2

```diff
@@ -2,3 +2,10 @@
     reverse_proxy localhost:7007
     tls infra@3box.io
 }
+
+{% if gitcoin_js_ceramic_hostname is defined and gitcoin_js_ceramic_hostname and 'js-ceramic' in gitcoin_js_ceramic_hostname %}
```

Comment:
I think a better way is to make the role use vars for the hostname and port/address of the reverse proxy.

Then use https://docs.ansible.com/ansible/latest/collections/ansible/builtin/blockinfile_module.html#ansible-collections-ansible-builtin-blockinfile-module module to just ensure the the section in the Caddyfile instead of trying to template the whole things.

Then the role could be called multiple times in a playbook.

Fine making this a TODO.

Reply:
This is for the generic URL `gitcoin-js-ceramic` and not `gitcoin-js-ceramic-3` (for the specific GC host). I was thinking it could actually be set on all the Ceramic hosts but would only be used on the host that the DNS resolves to (currently GC3). WDYT?

Reply:
Yeah, that sounds fine.